daemon/daemon_unix.go

```diff
@@ -111,6 +111,10 @@ func getMemoryResources(config containertypes.Resources) *specs.LinuxMemory {
 		memory.Kernel = &config.KernelMemory
 	}
 
+	if config.KernelMemoryTCP != 0 {
```

Comment:
If we anticipate adding this to `docker container update` as well, we need to think how to reset this